docs/debug.md

```diff
@@ -103,6 +103,57 @@ While pulling, the image manifest, config, and layers without zTOCs' are fetched
 
 Below are a list of common error paths that may occur in this phase:
 
+### Determining how an image was pulled
+
+The SOCI snapshotter can pull an image using one of four modes:
+1) Lazily using an explicitly provided SOCI index digest
+2) Lazily using a SOCI index manifest v2 via SOCI-enabled images
+3) Lazily using a SOCI index manifest v1 via the Referrers API
+4) Ahead of time without a SOCI index
+
+Determining which of these mode was used is often a good first step for debugging image pull issues.
+
+When SOCI starts to pull an image, it will emit a set of debug logs to indicate which modes were tried and which was ultimately used.
```

Comment:
There's a lot of mixing between the term SOCI and the term snapshotter. Do you think it would be better to explicitly say "SOCI snapshotter" in all cases?

Reply:
Yeah, that's a good callout. Will fix.